htdocs/core/class/html.formticket.class.php

```diff
@@ -883,7 +889,11 @@ public function clear_attached_files()
 			dol_delete_dir_recursive($upload_dir);
 		}
 
-		$keytoavoidconflict = empty($this->trackid) ? '' : '-'.$this->trackid; // this->trackid must be defined
+		if (!empty($this->trackid)) { // TODO Always use trackid (ticXXX) instead of track_id (abcd123)
+			$keytoavoidconflict = '-'.$this->trackid;
```

Comment:
trackid and track_id are 2 differents fields.
Only  ->trackid must be used for $keytoavoidconflict

Reply:
@eldy 
You're right. 
I corrected it and I only use track_id for key to avoid conflict.